<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchEngineClass</class>
 <widget class="QMainWindow" name="SearchEngineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SearchEngine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="documentsList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchLineEdit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ShowButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show best matching</string>
    </property>
   </widget>
   <widget class="QLabel" name="BestMatchingLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="FilterEdit">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>220</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SearchEngine.qrc"/>
 </resources>
 <connections/>
</ui>
